tests/TestHelpers/GraphHelper.php

```diff
@@ -524,6 +524,233 @@ public static function preparePatchUserPayload(
+	 * @param  array  $body
+	 * @param  array  $headers
+	 *
+	 * @return ResponseInterface
+	 *
+	 * @throws GuzzleException
+	 */
+	public static function getMySpaces(
+		string $baseUrl,
+		string $user,
+		string $password,
+		string $urlArguments = '',
+		string $xRequestId = '',
+		array  $body = [],
+		array  $headers = []
+	): ResponseInterface {
+		$url = self::getFullUrl($baseUrl, 'me/drives/' . $urlArguments);
+
+		return HttpRequestHelper::get($url, $xRequestId, $user, $password, $headers, $body);
+	}
+
+	/**
+	 * Send Graph List All Spaces Request
+	 *
+	 * @param  string $baseUrl
+	 * @param  string $user
+	 * @param  string $password
+	 * @param  string $urlArguments
+	 * @param  string $xRequestId
+	 * @param  array  $body
+	 * @param  array  $headers
+	 *
+	 * @return ResponseInterface
+	 *
+	 * @throws GuzzleException
+	 */
+	public static function getAllSpaces(
+		string $baseUrl,
+		string $user,
+		string $password,
+		string $urlArguments = '',
+		string $xRequestId = '',
+		array  $body = [],
+		array  $headers = []
+	): ResponseInterface {
+		$url = self::getFullUrl($baseUrl, 'drives/' . $urlArguments);
+
+		return HttpRequestHelper::get($url, $xRequestId, $user, $password, $headers, $body);
+	}
+
+	/**
+	 * Send Graph List Single Space Request
+	 *
+	 * @param string $baseUrl
+	 * @param string $user
+	 * @param string $password
+	 * @param string $spaceId
+	 * @param string $urlArguments
+	 * @param string $xRequestId
+	 * @param array $body
+	 * @param array $headers
+	 *
+	 * @return ResponseInterface
+	 *
+	 */
+	public static function getSingleSpace(
+		string $baseUrl,
+		string $user,
+		string $password,
+		string $spaceId,
+		string $urlArguments = '',
+		string $xRequestId = '',
+		array  $body = [],
+		array  $headers = []
+	): ResponseInterface {
+		$url = self::getFullUrl($baseUrl, 'drives/' . $spaceId . "/" . $urlArguments);
+
+		return HttpRequestHelper::get($url, $xRequestId, $user, $password, $headers, $body);
+	}
+
+	/**
+	 * send disable space request
+	 * 
+	 * @param string $baseUrl
+	 * @param string $user
+	 * @param string $password
+	 * @param string $spaceId
+	 * @param string $xRequestId
+	 * 
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function disableSpace(
+		string $baseUrl,
+		string $user,
+		string $password,
+		string $spaceId,
+		string $xRequestId = ''
+	): ResponseInterface {
+		$url = self::getFullUrl($baseUrl, 'drives/' . $spaceId);
+		
+		return HttpRequestHelper::delete(
+			$url,
+			$xRequestId,
+			$user,
+			$password
+		);
+	}
+
+	/**
+	 * send delete space request
+	 * 
+	 * @param string $baseUrl
+	 * @param string $user
+	 * @param string $password
+	 * @param string $spaceId
+	 * @param array  $header
+	 * @param string $xRequestId
+	 * 
+	 * @return ResponseInterface
+	 * @throws GuzzleException
+	 */
+	public static function deleteSpace(
+		string $baseUrl,
+		string $user,
+		string $password,
+		string $spaceId,
+		array $header,
```

Comment:
I think that the header is always:
`$header = ["Purge" => "T"];`

That code could be put in this function, similar to what is done in `restoreSpace` below.

But that can be done in a follow-up PR.

Reply:
Done in #4502